src/gavel2html.coffee

```diff
@@ -77,26 +82,26 @@ class Gavel2Html
   #@private
   getConverter: ->
     options = {
+      @fieldName
       @dataReal
       @dataExpected
-      @gavelResult
+      @fieldResult
       @usePointers
     }
 
-    switch @gavelResult?.validator
-      when 'TextDiff'
-        return new TextResultConverter options
+    switch @fieldResult?.kind
+      when 'json'
```

Comment:
what's the purpose of the switch now? there's just one `when` left, so it can be done with `if/else`

Reply:
Agree. Done in 468ac0c.